controllers/tests/mocks/scale.go

```diff
@@ -295,6 +295,21 @@ func (mr *MockCruiseControlScalerMockRecorder) Status(ctx interface{}) *gomock.C
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Status", reflect.TypeOf((*MockCruiseControlScaler)(nil).Status), ctx)
 }
 
+// StatusTask mocks base method.
```

Comment:
re-run `make mock-generate` to generate this file

Reply:
done, generated the file in 193a716